app/javascript/app/components/tab-switcher/tab-switcher-component.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Switch, ButtonGroup, Button, Icon, Input } from 'cw-components';
+import ModalMetadata from 'components/modal-metadata';
+
+import iconInfo from 'assets/icons/info';
+import downloadIcon from 'assets/icons/download';
+
+import styles from './tab-switcher-styles';
+
+class TabSwitcher extends PureComponent {
+  render() {
+    const {
+      tabs,
+      activeTabValue,
+      onTabChange,
+      onInfoClick,
+      onDownloadClick,
+      onFilterChange
+    } = this.props;
+    const activeTab = activeTabValue
+      ? tabs.find(t => t.value === activeTabValue)
+      : tabs[0];
+
+    return (
+      <div className={styles.wrapper}>
+        <div className={styles.toolbar}>
+          <Switch
+            options={tabs.map(o => ({ name: o.name, value: o.value }))}
+            selectedOption={activeTab.value}
+            onClick={onTabChange}
+            theme={{
+              wrapper: styles.switch,
+              option: styles.switchOption,
+              checkedOption: styles.switchSelected
+            }}
+          />
+          <div className={styles.toolbarActions}>
+            <ButtonGroup theme={{ wrapper: styles.buttonWrapper }}>
```

Comment:
this ButtonGroup with info and download icon is appearing in many designs so I moved it to another component within this task: https://www.pivotaltracker.com/story/show/159885649 
Once that story is ready we can use it here as well